tests/unit/domain/inventory/test_inventory_aggregate.py

```
@@ -0,0 +1,174 @@
+    def test_allocate_negative_quantity_raises(self):
+        """Test that allocating negative quantity raises ValueError."""
+        aggregate = InventoryAggregate(
+            item_id="ITEM001",
+            quantity=100,
+            reserved_quantity=0,
+            location="WAREHOUSE_A"
+        )
+        with pytest.raises(ValueError, match="Allocation quantity must be positive"):
+            aggregate.allocate(quantity=-10)
+
+    def test_allocate_zero_quantity_raises(self):
```

Ruff (0.12.2)

Redefinition of unused `test_allocate_zero_quantity_raises` from line 10

(F811)